src/enums/data-location.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-unused-vars */
+export enum DataLocation {
+    BLOB_STORAGE = 'BlobStorage',
+    DATA_LAKE = 'Datalake'
+}
```

Comment:
Moved some of the enums to their own files - still need to investigate what the deal is with eslint hating on them.

Reply:
So the answer as to why eslint dislike enums is that they're not native to JavaScript and when the transpiler converts the code it does some screwy stuff to make enums work.

Reply:
Yeah but I've used enums in typescript tons before without eslint crying - I think we just need to fiddle with the eslint config.